packages/typescript-plugin/src/language-service/index.ts

```diff
@@ -64,11 +69,6 @@ function createProxyHandler(configManager: ConfigManager): ProxyHandler<ts.Langu
 
     return {
         get(target, p) {
-            // always return patch symbol whether the plugin is enabled or not
-            if (p === sveltePluginPatchSymbol) {
-                return true;
-            }
-
```

Comment:
Is the `if (!configManager.getConfig().enable || p === 'dispose') {` below still correct?

Reply:
Hmm. I don't remember why `dispose` has special treatment here. But it seems wrong.

Reply:
I remember why `dispose` has special treatment here now. It should run the decorated version even if the typescript-plugin is disabled, but the current check is wrong.